File: Admin/src/main/resources/static/staff/js/customer.js
async function findAllCustomer() {
  $('#example').DataTable().destroy();
  var url = 'http://localhost:8080/staff/all-customer';
  const response = await fetch(url, {
    method: 'GET'
  });
  var list = await response.json();
  var main = '';
  for (i = 0; i < list.length; i++) {
    main += `<tr>
                  <td>${list[i].id}</td>
                  <td><img src="${list[i].avatar}" style="width: 100px"></td>
                  <td>${list[i].address}</td>
                  <td>${list[i].email}</td>
                  <td>${list[i].fullName}</td>
                  <td>${list[i].phone}</td>
                  <td>${list[i].username}</td>
              </tr>`
  }
  document.getElementById("listCustomer").innerHTML = main;
  $('#example').DataTable();
}

File: Customer/src/main/java/com/example/customer/domain/Customer.java
package com.example.customer.domain;


import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class Customer {

    private Long id;

    private String username;

    private String password;

    private String avatar;

    private String phone;

    private String fullName;

    private String address;

    private String email;


    private List<Order> orders;

    private List<Review> reviews;
}
